Source/WebKit/Shared/API/Cocoa/WKRemoteObjectCoder.mm

```diff
@@ -984,13 +984,13 @@ static void decodeInvocationArguments(WKRemoteObjectDecoder *decoder, NSInvocati
     return invocation;
 }
 
-static NSString *decodeString(WKRemoteObjectDecoder *decoder)
+static RetainPtr<NSString> decodeString(WKRemoteObjectDecoder *decoder)
 {
     API::String* string = decoder->_currentDictionary->get<API::String>(stringKey);
     if (!string)
         [NSException raise:NSInvalidUnarchiveOperationException format:@"String missing"];
 
-    return string->string();
```

Comment:
It is really non-obvious to me that this function makes an isolated copy!

Reply:
Indeed, but:
`    WTF::String string() const { return m_string.isolatedCopy(); }`

Maybe we can rename it to isolatedString() and double-check if all call sites actually need the isolated copy.